### ISSUE TITLE
All the benchmarks are bitrotted

### ISSUE THREAD

**Issue by enolan** [idris-lang/Idris-dev#2819]
None of them compile. We should really make them part of the test suite. Here's the output when trying to build them:

```
enolan at Behemoth in ~/mystuff/code/Idris-dev/benchmarks [float-double*]
$ ./build.pl
Building trivial / sortvec
Removing sortvec.ibc
WARNING: Cannot remove file sortvec.ibc, Error msg:sortvec.ibc: removeLink: does not exist (No such file or directory)
Removing /home/enolan/mystuff/code/Idris-dev/benchmarks/trivial/sortvec
WARNING: Cannot remove file /home/enolan/mystuff/code/Idris-dev/benchmarks/trivial/sortvec, Error msg:/home/enolan/mystuff/code/Idris-dev/benchmarks/trivial/sortvec: removeLink: does not exist (No such file or directory)
Type checking ./sortvec.idr
sortvec.idr:7:8:When checking type of Main.insert:
No such variable Vect
Building quasigroups / qgsolve
Removing Solver.ibc
WARNING: Cannot remove file Solver.ibc, Error msg:Solver.ibc: removeLink: does not exist (No such file or directory)
Removing Parser.ibc
WARNING: Cannot remove file Parser.ibc, Error msg:Parser.ibc: removeLink: does not exist (No such file or directory)
Removing Main.ibc
WARNING: Cannot remove file Main.ibc, Error msg:Main.ibc: removeLink: does not exist (No such file or directory)
Removing /home/enolan/mystuff/code/Idris-dev/benchmarks/quasigroups/qgsolve
WARNING: Cannot remove file /home/enolan/mystuff/code/Idris-dev/benchmarks/quasigroups/qgsolve, Error msg:/home/enolan/mystuff/code/Idris-dev/benchmarks/quasigroups/qgsolve: removeLink: does not exist (No such file or directory)
Type checking ./Solver.idr
Solver.idr:10:6:
When checking right hand side of Cell with expected type
        Type

When checking an application of type constructor Prelude.Maybe.Maybe:
        No such variable Fin
Solver.idr:13:11:When checking type of Solver.MkBoard:
No such variable Vect
Solver.idr:15:12:When checking type of Solver.emptyBoard:
No such variable Board
Building fasta / fasta
Removing fasta.ibc
WARNING: Cannot remove file fasta.ibc, Error msg:fasta.ibc: removeLink: does not exist (No such file or directory)
Removing /home/enolan/mystuff/code/Idris-dev/benchmarks/fasta/fasta
WARNING: Cannot remove file /home/enolan/mystuff/code/Idris-dev/benchmarks/fasta/fasta, Error msg:/home/enolan/mystuff/code/Idris-dev/benchmarks/fasta/fasta: removeLink: does not exist (No such file or directory)
Can't find import Data/Floats
Building pidigits / pidigits
Removing pidigits.ibc
WARNING: Cannot remove file pidigits.ibc, Error msg:pidigits.ibc: removeLink: does not exist (No such file or directory)
Removing /home/enolan/mystuff/code/Idris-dev/benchmarks/pidigits/pidigits
WARNING: Cannot remove file /home/enolan/mystuff/code/Idris-dev/benchmarks/pidigits/pidigits, Error msg:/home/enolan/mystuff/code/Idris-dev/benchmarks/pidigits/pidigits: removeLink: does not exist (No such file or directory)
Type checking ./pidigits.idr
pidigits.idr:15:14:
Main.mkF has a name which may be implicitly bound.
This is likely to lead to problems!
pidigits.idr:19:6:
When checking right hand side of loop with expected type
        IO ()

When checking an application of function Prelude.Interactive.putStrLn:
        Can't disambiguate since no name has a suitable type:
                Prelude.List.++, Prelude.Strings.++
```

**Comment by LeifW** [comment]
That, or perhaps have a secondary job on travis, that is kicked off as a result of the primary job (depends on the idris binary produced) to run these?

**Comment by LeifW** [comment]
Was thinking of having peoples various idris programs / libs hooked in in CI that way, too. So we can see when they get broken...

**Comment by enolan** [comment]
Fixed in 79ff02f :)
